build.sbt

```diff
@@ -66,10 +66,13 @@ lazy val root = (project in file(".")).
 
 lazy val akkaHttpVersion = "10.0.11"
 
+lazy val configVersion = "1.3.1"
+
 lazy val `akka-http-cors` = project.
   settings(commonSettings: _*).
   settings(publishSettings: _*).
   settings(
+    libraryDependencies += "com.typesafe" % "config" % configVersion,
```

Comment:
Akka has already this dependency, we don't want to override their version of `config`.